lib/text-class/test/unit/Data/Text/ClassSpec.hs

```diff
@@ -31,7 +31,7 @@ spec = do
         it "toText -14" $
             toText @Int (-14) === "-14"
         it "fromText \"patate\"" $
-            let err = "input does not start with a digit"
+            let err = "Int is an integer number between -9223372036854775808 and 9223372036854775807."
```

Comment:
We should probably use `minBound` and `maxBound` to generate this string, as the exact limits depend on the Haskell implementation.

Reply:
ah - yes that's true